xml/operations-configuring-identity-websso.xml

```diff
@@ -322,22 +322,24 @@
    </listitem>
    <listitem>
     <para>
-     Copy adfs_metadata.xml to the &clm; node in your preferred
-     location. Here it is /tmp.
+     Copy adfs_metadata.xml to the &clm; node and place it into <filename>/var/lib/ardana/openstack/my_cloud/config/keystone/</filename> directory and put it under revision control.
     </para>
+<screen>&prompt.ardana;cd ~/openstack
+&prompt.ardana;git checkout site
+&prompt.ardana;git add my_cloud/config/keystone/adfs_metadata.xml
+&prompt.ardana;git commit -m "Add ADFS metadata file for WebSSO authentication"
```

Comment:
think you're missing the `</screen>` closing tag here

Reply:
good catch!